.storybook/preview.js

```diff
@@ -129,8 +129,8 @@ const sbFrameReset=(Story,context)=>{
   sidebarItem.forEach(function(item, i){
     item.addEventListener('click', function (e) {
       // Remove body class only if it exists.
-      if (iframeBody.classList.contains('sdgmodal-open')) {
-        iframeBody.classList.remove('sdgmodal-open');
+      if (iframeBody.classList.contains('sdgmodal-open'),('color-blue')) {
+        iframeBody.classList.remove('sdgmodal-open','color-blue');
```

Comment:
@Shubham-Sharma-77 This is not the correct way to use `classList.contains()`. Kindly change the above lines to:
```
// Classes to remove.
const classNames = ['sdgmodal-open', 'color-blue'];
// Check if above classes exist in `body` element and remove them.
if (classNames.some(className => iframeBody.classList.contains(className))) {
  iframeBody.classList.remove(...classNames);
}
// This will remove the `'sdgmodal-open', 'color-blue'` classes respectively.
```

Reply:
@shishirphadke Ok